<launch>

    <param name="user_name" type="string" value="default_player"/>
    <param name="change_player_color" type="int" value="1"/>
    <param name="screen_param" type="string" value="phase1"/>

    
    <node name="result_node" 
          pkg="ros_game" 
          type="result_node.py" 
          output="screen"/>
    
    <node name="game_node" 
          pkg="ros_game" 
          type="game_node.py" 
          output="screen"/>
    
    <node name="control_node_pygame" 
          pkg="ros_game" 
          type="control_node_pygame.py" 
          output="screen"/>
    
    <!--node name="info_user" 
          pkg="ros_game" 
          type="info_user.py"
          output="screen"/-->

    <!--node name="control_node" pkg="ros_game" type="control_node.py" output="screen"/-->
</launch>